js/cmb2.js

```diff
@@ -695,24 +695,30 @@ window.CMB2 = window.CMB2 || {};
 	cmb.removeGroupRow = function( evt ) {
 		evt.preventDefault();
 
-		var $this   = $( this );
-		var $table  = $id( $this.data('selector') );
-		var $parent = $this.parents('.cmb-repeatable-grouping');
-		var number  = $table.find('.cmb-repeatable-grouping').length;
+		var $this         = $( this ),
+			isConfirmable = $this.data('confirm') === 'yes';
```

Comment:
Please go back to the separate var statements-per-line as that is the code style of this lib. ([reasons](https://danhough.com/blog/single-var-pattern-rant/))

Reply:
Older articles, but I see the point. Will update, thanks for the links!